doc-content/enterprise-only/openshift/rhpam73-authoring-ref.adoc

```diff
@@ -47,9 +48,13 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_HTTPS_NAME` | `HTTPS_NAME` | The name associated with the server certificate | jboss | False
 |`KIE_SERVER_HTTPS_PASSWORD` | `HTTPS_PASSWORD` | The password for the keystore and certificate | mykeystorepass | False
 |`DB_VOLUME_CAPACITY` | -- | Size of persistent storage for the database volume. | 1Gi | True
+|`KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | `KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property) | false | False
+|`KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | `KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | KIE ServerTemplate Cache TTL in milliseconds (Sets the org.kie.server.controller.template.cache.ttl system property) | 60000 | False
+|`KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` | `KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` | If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property) | false | False
 |`IMAGE_STREAM_NAMESPACE` | -- | Namespace in which the ImageStreams for Red Hat Middleware images are installed. These ImageStreams are normally installed in the openshift namespace. You should only need to modify this if you installed the ImageStreams in a different namespace/project. | openshift | True
-|`KIE_SERVER_IMAGE_STREAM_NAME` | -- | The name of the image stream to use for KIE server. Default is "rhpam72-kieserver-openshift". | rhpam72-kieserver-openshift | True
-|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.1". | 1.1 | True
+|`KIE_SERVER_IMAGE_STREAM_NAME` | -- | The name of the image stream to use for KIE server. Default is "rhpam73-kieserver-openshift". | rhpam73-kieserver-openshift | True
+|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.0". | 1.0 | True
+|`MAVEN_MIRROR_URL` | `MAVEN_MIRROR_URL` | Maven mirror to use by Business Central and KIE server. | -- | False
```

Comment:
s/Maven mirror to use by Business Central and KIE server/Maven mirror to use for Business Central and KIE server